scripts/hubspot-deploy/hubspot-api.js

```diff
@@ -86,7 +93,7 @@ class HubSpotApi {
   }
 
   async deletePage(pageId){
-    const url = `${this.baseUrl}/${pageId}?hapikey=${this.apiKey}`;
+    const url = `/${pageId}`;
```

Comment:
I like that the apiKey has been removed on these, it makes sense that the auth parts would be handled in one place. 👍